src/services/request/BaseRequest.class.spec.js

```diff
@@ -50,7 +50,7 @@ describe('RequestFactory', () => {
 
     const expectedAuthorizationHeader = Buffer.from('username:password').toString('base64')
     const expectedHeader = { Authorization: `Basic ${expectedAuthorizationHeader}` }
-    expect(result).toEqual(ApiHandler.STATUS.SUCCESS)
+    expect(result).toEqual(NorthHandler.STATUS.SUCCESS)
```

Comment:
I think these STATUS should not be in the NorthHandler file : the same status can be used when sending logs or health signal fails